packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -442,6 +444,7 @@ const ToggleMenuDrawer = observer(({ platform_config }) => {
                     onClick={() => {
                         if (!is_language_change) {
                             setIsLanguageChange(true);
+                            setIsPlatformSwitcherOpen(false);
```

Comment:
The fix for mobile language switcher: Forcing platform switcher to close if the language switcher is open.